resources/views/components/partial/police-accountability/chart-use-of-force.blade.php

```diff
@@ -11,7 +11,7 @@
     <p>
         {{ num($scorecard['police_accountability']['use_of_force_complaints_reported']) }} Reported
 
-        @if($scorecard['report']['percent_use_of_force_complaints_sustained'])
+        @if(isset($scorecard['report']['percent_use_of_force_complaints_sustained']))
```

Comment:
Extract into variable since it's used many times.

Reply:
It already is a variable `$scorecard` is loaded in the route and that same variable is passed through all the partial templates.  That variable is just a multidimensional array that contains all the data needed. There are several hundred places in this project we would need to make out custom template variables if we went this route, so do not make sense to do this for just this PR.

Reply:
"$scorecard['report']['percent_use_of_force_complaints_sustained']" this variable is used multiple times. The number of times we index into the map outweighs the memory alloc and lack of readability.

Reply:
right, my point though is that we are doing this ALL OVER this project, just just here.  If we want to tweak this, which I am fine with, we should make this a dedicated ticket and redo ALL instances of where this happens, not just this one line in this PR where it happens to be happening.

If there are performance gains to streamline this, it should be done site wide as a dedicated ticket, as just doing it for this one instance kind of does not make sense IMO when the rest of the code base has the exact same issues.

Totally fine knocking this out on my end, but just don't think it should block this specific PR from going out.

Reply:
Good point. Let's refactor in a separate change.